enterprise/templatetags/enterprise.py

```diff
@@ -0,0 +1,30 @@
+"""
+Template tags for enterprise apps.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()  # pylint: disable=invalid-name
+
+
+@register.simple_tag(name="fa_icon")
+def fa_icon(tags):
+    """
+    Django template tag that returns font awesome icon depending upon message tags.
+
+    Usage:
+        {% fa_icon "success" %}
+    """
+    if 'success' in tags:
```

Comment:
@zubair-arbi I do not think that would be a good idea, as we should only be displaying one icon per message. i.e. a message can either be success or warning but not both, same goes for other combinations for success, info, warning and error.